src/NuGetTrends.Web/Startup.cs

```diff
@@ -111,6 +107,9 @@ public void Configure(IApplicationBuilder app)
                 });
             }
 
+            // Proxy Sentry events from the frontend to sentry.io
+            app.UseSentryTunneling("/t");
```

Comment:
Positioning after CORS middleware
It was capturing the events but showing CORS errors on the browser (and acting up due to that).